### DISCUSSION TITLE
“真空断路器” 导致了 SyncPlay 文件名传递异常

### DISCUSSION THREAD

**Discussion by txyyh** [hooke007/mpv_PlayKit#319] [mpv 通用]
算不上是个 bug，主要还是 SyncPlay 数据传递的实现方式导致的。
SyncPlay 通过 mpv 的 log 传递文件名等信息。他会先在 log 中输出 term-msg，之后通过读取 log 来实现数据传递。
而真空断路器输出的 msg 会覆盖 SyncPlay 输出的 msg， 导致文件名无法被传递，表现为无法识别正在播放的文件。

使用  SyncPlay 的可以留意下，尽量避免输出 term-msg 吧。

**Comment by hooke007** [comment]
这属于syncplay的设计缺陷吧，这种完全依赖mpv某个选项的程序都有类似的问题。比如 svp manager 需要强迫用户不自行使用 input-ipc-server 作其它用途
